service_metadata.json

```diff
@@ -1,26 +1,125 @@
 {
   "version": 1,
-  "display_name": "Example1",
+  "display_name": "PRICING MODEL",
   "encoding": "proto",
   "service_type": "grpc",
-  "payment_expiration_threshold": 40320,
-  "model_ipfs_hash": "QmUKGT6n8NUey2BQ9fqfrYZRcktZjZ1rDj6Qe7wc4o7Lwh",
-  "mpe_address": "0x27CFC34Aa127E4fb2Bee50563F15489fea20dA58",
-  "pricing": {
-    "price_model": "fixed_price",
-    "price_in_cogs": 10000000
-  },
+
+  "model_ipfs_hash": "Qmdiq8Hu6dYiwp712GtnbBxagyfYyvUY1HYqkH7iN76UCc",
+  "mpe_address": "0x7E6366Fbe3bdfCE3C906667911FC5237Cc96BD08",
   "groups": [
     {
+      "endpoints": ["http://34.344.33.1:2379","http://34.344.33.1:2389"],
```

Comment:
This is a sample metadata file? This is used when the daemon is started with blockchain disabled right? In that case will this not break as it has method level pricing set up assuming ExampleService

Reply:
we dont need sample metadata any more